src/controllers/errorsController.js

```diff
@@ -51,13 +63,14 @@ class ErrorsController extends PageController {
         }
 
         if (entryNumber in aggregatedIssues) {
-          const columnName = this.lookupMappedColumnNameFromOriginal(issue.field, apiResponseData['column-field-log'])
-          aggregatedIssues[entryNumber][columnName] = {
+          const columnName = this.lookupOriginalColumnNameFromMapped(issue.field, apiResponseData['column-field-log'])
+
+          aggregatedIssues[entryNumber][issue.field] = {
             issue: {
               type: issue['issue-type'],
               description: issue.description
             },
-            value: rowValues[issue.field]
+            value: rowValues[columnName]
```

Comment:
Explainer: this was previously incorrect, and causing issues from fields we want to ignore to overwrite those that we want to keep